route/battle_test/battle_test.py

```diff
@@ -0,0 +1,32 @@
+from engine.combat import SeqCombatAndMove
+from engine.seq import (
+    InteractMove,
+    SeqDelay,
+    SeqList,
+)
+
+
+class BattleTest(SeqList):
+    def __init__(self):
+        super().__init__(
+            name="BattleTest",
+            children=[
+                SeqDelay(name="Wait for select screen", timeout_in_s=3.0),
+                BattleTestSequence(),
+            ],
+        )
+
+
+class BattleTestSequence(SeqList):
```

Comment:
Possibly add a comment here about which map the code is intended to be run on.

Reply:
Any! Its just to start the combat controller

Reply:
correct - i'll make some notes to make it clearer what its used for.

It's simply so i can start combat sequence whenever i want to test new changes; without messing with other sequences